process_images.sh

```diff
@@ -67,6 +69,9 @@ EOF
       --ocr-subdir) RESULT_SUBDIR="$2"; shift;;
       --proc-id) PROCESS_ID="$2"; shift;;
       --task-id) TASK_ID="$2"; shift;;
+      --async) ASYNC="$2"; shift;;
```

Comment:
strongly recommend against making this parameter take an argument
```suggestion
      --async) ASYNC=true;;
```